drivers/bridge/bridge.go

```diff
@@ -108,9 +109,7 @@ func newDriver() driverapi.Driver {
 func Init(dc driverapi.DriverCallback) error {
 	// try to modprobe bridge first
 	// see gh#12177
-	if out, err := exec.Command("modprobe", "-va", "bridge", "nf_nat", "br_netfilter").CombinedOutput(); err != nil {
-		logrus.Warnf("Running modprobe bridge nf_nat br_netfilter failed with message: %s, error: %v", out, err)
-	}
+	out, err1 := exec.Command("modprobe", "-va", "bridge", "nf_nat", "br_netfilter").CombinedOutput()
```

Comment:
I understand from docker/docker#13981 & docker/docker#13980 that the attempt was to change Output() to CombinedOutput(). But as of https://github.com/docker/libnetwork/pull/304 that was taken care of.

Given that, I don't see how this particular change will be helpful ?
Especially to throw this warning when RegisterDriver fails doesn't seem right.